core/config/app_config.go

```diff
@@ -54,6 +54,7 @@ type AppConfig interface {
 	Threshold() Threshold
 	WebServer() WebServer
 	Tracing() Tracing
+	Transmission() Transmission
```

Comment:
What code will call this?

Reply:
I have to figure out how to wire this through Mercury (`.../relay/evm/mercury/transmitter.go` + `.../relay/evm/mercury/wsrpc/clientgo` ), but eventually it'll end up in a call to
https://github.com/grpc/grpc-go/blob/v1.59.0/credentials/tls.go#L168

To better protect these changes while things shake out, I can open PRs against a feature branch. That might be best from a release perspective too while we stage these changes? Other option is a feature flag to switch between wsrpc and grpc

Reply:
Would it make sense to move this underneath `Mercury`? e.g. `Mercury.Tranmission.TLS`? or just `Mercury.TLS`?

Reply:
After *a lot* of back and forth, I think it's the right call to move this underneath Mercury, since right now the only config field we need to drill is the cert bundle. Another config parameter I debated including was the minimum acceptable TLS version, but that also doesn't need to be core specific - it feels acceptable to allow CLO RTSP and Gateway to make individuals calls on that.